exonum/src/blockchain/mod.rs

```diff
@@ -145,7 +145,7 @@ impl Blockchain {
 
     /// Starts promotion into a mutable blockchain instance that can be used to process
     /// transactions and create blocks.
-    pub fn into_mut(self, genesis_config: ConsensusConfig) -> BlockchainBuilder {
+    pub(crate) fn into_mut(self, genesis_config: ConsensusConfig) -> BlockchainBuilder {
```

Comment:
Well, it's sort of helpful for tests, since it allows us to have a pretty builder pattern, [like here](https://github.com/exonum/exonum/blob/dynamic_services/exonum/src/runtime/rust/tests.rs#L303).

So I think I'll leave it but put under `#[cfg(test)]`.